app/models/shipit/user.rb

```diff
@@ -140,8 +140,9 @@ def appropriate_email_for(github_user)
                   .sort_by { |e| e.primary ? 0 : 1 }
                   .map(&:email)
                   .find { |e| email_valid_and_preferred?(e) }
-      rescue Octokit::NotFound, Octokit::Forbidden
+      rescue Octokit::NotFound, Octokit::Forbidden, Octokit::Unauthorized
         # If the user hasn't agreed to the necessary permission, we can't access their private emails.
+        Rails.logger.warn("Failed to retrieve emails for user '#{name}'")
```

Comment:
`login` or `id` will be more stable I reckon.

Reply:
I realized that it makes more sense to use the github_user that was passed in, so I've deferred to that and am now using the same name logic as is used in the record setter elsewhere.